spec/webmock_spec.cr

```diff
@@ -329,6 +368,50 @@ describe WebMock do
     end
   end
 
+  it "contains stubbing instructions on failure (with HTTP:Request and no path)" do
+    WebMock.wrap do
+      begin
+        url = "https://www.example.com"
+        request = HTTP::Request.new("GET", url)
+        uri = URI.parse(url)
+        HTTP::Client.new(uri).exec(request)
+      rescue ex : WebMock::NetConnectNotAllowedError
+        ex.message.not_nil!.strip.should eq(
+          <<-MSG
+          Real HTTP connections are disabled. Unregistered request: GET https://www.example.com with headers {"Host" => "www.example.com"}
+
+          You can stub this request with the following snippet:
+
+          WebMock.stub(:get, "https://www.example.com").
+            to_return(body: "")
+          MSG
+        )
+      end
+    end
+  end
+
+  it "contains stubbing instructions on failure (with HTTP:Request and path)" do
+    WebMock.wrap do
+      begin
+        url = "https://www.example.com/test"
+        request = HTTP::Request.new("GET", url)
+        uri = URI.parse(url)
+        HTTP::Client.new(uri).exec(request)
+      rescue ex : WebMock::NetConnectNotAllowedError
```

Comment:
Instead of `begin ... rescue` I think you should use `expect_raises`. ditto below.

Reply:
Mostly because I wasn't aware of it existing in Crystal 😃  Can fix

Reply:
Done @Sija